lib/customerio/api.rb

```diff
@@ -0,0 +1,23 @@
+require 'net/http'
+require 'multi_json'
+
+module Customerio
+  class ApiClient
+    DEFAULT_BASE_URI = 'https://api.customer.io'
+
+    def initialize(app_key, options = {})
+      options[:base_uri] = DEFAULT_BASE_URI if options[:base_uri].nil? || options[:base_uri].empty?
+      @client = Customerio::BaseClient.new({ app_key: app_key }, options)
+    end
+
+    def send_email(payload)
+      @client.request(:post, send_email_path, payload)
```

Comment:
track endpoints wrap calls with `verify_response` any particular reason to not perform that check here?

Reply:
Good point! No reason to not do it, so I've moved `verify_response` into the base client and I'm now calling that method here.